cfgmgr/vlanmgr.cpp

```diff
@@ -51,15 +51,17 @@ VlanMgr::VlanMgr(DBConnector *cfgDb, DBConnector *appDb, DBConnector *stateDb, c
     // The command should be generated as:
     // /bin/bash -c "/sbin/ip link del Bridge 2>/dev/null ;
     //               /sbin/ip link add Bridge up type bridge &&
+    //               /sbin/ip link set Bridge mtu {{ mtu_size }} &&
     //               /sbin/bridge vlan del vid 1 dev Bridge self;
     //               /sbin/ip link del dummy 2>/dev/null;
     //               /sbin/ip link add dummy type dummy &&
-    //               sbin/ip link set dummy master Bridge"
+    //               /sbin/ip link set dummy master Bridge"
 
     const std::string cmds = std::string("")
       + BASH_CMD + " -c \""
       + IP_CMD + " link del " + DOT1Q_BRIDGE_NAME + " 2>/dev/null; "
       + IP_CMD + " link add " + DOT1Q_BRIDGE_NAME + " up type bridge && "
+      + IP_CMD + " link set " + DOT1Q_BRIDGE_NAME + " mtu " + DEFAULT_MTU_STR + " && "
```

Comment:
If I remember correctly, the issue is the Bridge mtu setting is not consistent. If there is an interface thats enslaved has a lower mtu, say 1500, like the dummy interface or before we set the FP interface to 9100, then it resets the Bridge mtu. You could check comments [here ](https://github.com/Azure/sonic-swss/blob/master/cfgmgr/vlanmgr.cpp#L307)

Reply:
I just build an image and verified that Vlan1000 does have MTU set as 9100 with this change.

Reply:
Another PR - https://github.com/Azure/sonic-swss/pull/1161